Examples/GetValueHttp/GetValueHttp.cpp

```diff
@@ -0,0 +1,43 @@
+// This example retrieves last value of a variable from the Ubidots API
+// using HTTP protocol.
+
+/****************************************
+ * Include Libraries
+ ****************************************/
+
+#include "Ubidots.h"
+
+/****************************************
+ * Define Constants
+ ****************************************/
+
+const char* UBIDOTS_TOKEN = "...";  // Put here your Ubidots TOKEN
+
+Ubidots ubidots(UBIDOTS_TOKEN, UBI_HTTP);
+
+/****************************************
+ * Auxiliar Functions
+ ****************************************/
+
+// Put here your auxiliar functions
+
+/****************************************
+ * Main Functions
+ ****************************************/
+
+void setup() {
+  Serial.begin(115200);
+  // ubidots.setDebug(true); //Uncomment this line for printing debug messages
+}
+
+void loop() {
+  /* Obtain last value from a variable as float using HTTP */
+  float value = ubidots.get("weather-station", "temperature");
+
+  // Evaluates the results obtained
+  if (value != ERROR_VALUE) {
+    Serial.print("Value:");
+    Serial.println(value);
+  }
+  delay(5000);
```

Comment:
this kind of implementation must be done by the user rather than for us, please remember that we just expose methods for sending data but the logic to use them is at the user side.